### DISCUSSION TITLE
ERROR: failed to export: determining entrypoint: tried to set web to default but it doesn't exist

### DISCUSSION THREAD

**Discussion by Sunny-001** [paketo-buildpacks/feedback#56] [General]
i want to build a image for a vue project in tekton buildpacks, i tried version `0.3.106-base` and `base` of the builder image `paketobuildpacks/builder`,  i get an error in step `EXPORTING` :
![image](https://user-images.githubusercontent.com/46052043/203964557-22709796-63ee-4ee2-93d6-6179d56f6dc4.png)

**Comment by dmikusa** [comment]
It's not clear exactly what's happened, but by default, the endpoint used to start the image is called "web" so some buildpack needs to create a process type during the build for `web`. 

I'm not sure what all buildpacks you're using, so it's hard to say if that didn't work and the buildpack didn't create it but should have or if the buildpacks used just don't create it and thus you need to manually create it. You can also change the default. I'm not sure how to do that in Tekton, but with `pack build` there's an argument to change the default, so if you are legitimately just using a different named process type you can change the default to that.

At the end of the day, you either need a process type called `web` or to change the default to a process type you're generating. If the buildpacks are not doing this, you can add a `Procfile` and create a `web: <your process>` line which will make a process type called `web`. I would suggest you find out more about what is happening before doing this though, because buildpacks will usually take care of this for you.

If you could post the full output of your build as well as the list of buildpacks you're using, perhaps we can help some more.

**Reply by Sunny-001** [reply]
Thank you, I have solved this problem because the scripts module in package.json must contain start, which is not in my project。
```
  "scripts": {
    "start": "vue-cli-service serve",
  }
```
